src/Cloud/DecoratorPluginManager.php

```diff
@@ -21,26 +22,19 @@
  */
 class DecoratorPluginManager extends AbstractPluginManager
 {
-    /**
-     * Default set of decorators
-     *
-     * @var array
-     */
-    protected $invokableClasses = [
-        'htmlcloud' => 'Zend\Tag\Cloud\Decorator\HtmlCloud',
-        'htmltag'   => 'Zend\Tag\Cloud\Decorator\HtmlTag',
-        'tag'       => 'Zend\Tag\Cloud\Decorator\HtmlTag',
+    protected $aliases = [
+        'htmlcloud' => Decorator\HtmlCloud::class,
+        'htmltag'   => Decorator\HtmlTag::class,
+        'tag'       => Decorator\HtmlTag::class,
+    ];
+
+    protected $factories = [
+        Decorator\HtmlCloud::class => InvokableFactory::class,
+        Decorator\HtmlTag::class   => InvokableFactory::class,
     ];
 
     /**
-     * Validate the plugin
-     *
-     * Checks that the decorator loaded is an instance
-     * of Decorator\DecoratorInterface.
-     *
-     * @param  mixed $plugin
-     * @return void
-     * @throws Exception\InvalidArgumentException if invalid
+     * {@inheritdoc}
      */
     public function validatePlugin($plugin)
```

Comment:
Remove this method, and instead define an `$instanceOf` property with the value `Decorator\DecoratorInterface::class`.